tests/SpecTests/client-side-encryption/tests/fle2-BypassQueryAnalysis.json

```diff
@@ -4,7 +4,8 @@
       "minServerVersion": "6.0.0",
       "topology": [
         "replicaset",
-        "sharded"
+        "sharded",
+        "load-balanced"
```

Comment:
None of the commits in this PR indicate the sync target for spec tests. According to [DRIVERS-2400](https://jira.mongodb.org/browse/DRIVERS-2400) this is https://github.com/mongodb/specifications/commit/59a07b718253457d380f1e69cd51ad417f36e575. Please add that and ensure it ends up in the final squash message.

Reply:
Ah, yes. Added to the last commit and will make sure it ends in the squash message.